vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -7,11 +7,50 @@ sidebar_position: 1
 import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
 
 import ApiKey from '@site/platform/api/_partials/resources/config/external/platform/apiKey.mdx'
+import Flow, { Step } from '@site/src/components/Flow';
 
 <ProAdmonition/>
 
+<!--vale off-->
+
 This provides a way to connect directly to the vCluster Platform.
 
+## Connect vCluster to the platform 
+
+You can connect virtual clusters to the platform using a managed API key. 
+Use a shared Kubernetes secret to connect a vCluster to the platform. Create the secret in a namespace, then reference it in the `vcluster.yaml` file to enable authentication.  
```

Comment:
This should specify that the namespace needs to be on the host cluster where the virtual cluster is to be deployed, but doesn’t need to be in the same namespace of where the vcluster cluster control plane is deployed.